### PR TITLE
TCAF Sprite Update

### PR DESCRIPTION
**Pull request by Ben10083**
- TCAF sprites for (most) TCAF clothing items updated, credit to @NobleRow for the sprites
- new TCAF chest gear added for light armor chest rig, 2 slot storage
- telecrystal tcaf uplink crate explicitly set to 50
- Visor helmets and toeless jackboots added to TCAF uplink crates

### CODE REVIEW

**Comment by Ben10083** [comment]
Old 
image

New
image
image
image

**Comment by Ben10083** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]
Sure.

File: html/changelogs/Ben10083 - TCAF_Sprites.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - imageadd: "New sprites for most TCAF clothing, credit to Noble Row for the spritework!"
+  - rscadd: "New TCAF chest gear added for light armor variant of plate carrier."
+  - imageadd: "New sprites for most TCAF clothing, credit to Noble Row for the spritework!"
+  - bugfix: "uplink TCAF group crate cost set to 50 telecrystals. (it was not set in code)"
```
Comment (FluffyGhoster) [inline html/changelogs/Ben10083 - TCAF_Sprites.yml:44]:
```suggestion
  - bugfix: "Uplink TCAF group crate cost set to 50 telecrystals. (it was not set in code)"
```

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**